internal/pkg/manifest/svc.go

```diff
@@ -495,6 +495,46 @@ func (rr RoutingRuleConfiguration) exposedPorts(exposedPorts []ExposedPort, work
 	}
 }
 
+// exportPorts returns any new ports that should be exposed given the network load balancer
+// configuration that's not part of the existing containerPorts.
+func (cfg NetworkLoadBalancerConfiguration) exposedPorts(exposedPorts []ExposedPort, workloadName string) ([]ExposedPort, error) {
+	if cfg.IsEmpty() {
+		return nil, nil
+	}
+	containerName := workloadName
```

Comment:
I have a few question/nits:
1. Would `targetContainer` be an accurate name for `containerName`? Would `targetPort` be an accurate name for `port`? If so, can we rename the variables to `targetContainer` and `targetPort`?
2. If `containerName` is only used to add the port to `exposedPort` then can we parse its value toward the end, just before `return []ExposedPort{...}`? My reasoning is that this increases code readability. When I was reading the snippet, I kept wondering "what is `containerName` for" and didn't find out why until I reached the end 😂 

For example,
```go
if cfg.IsEmpty() {
  return nil, nil
}

// parse target port
// check if port is already exposed
// port isn't exposed yet, then...
targetContainer := workloadName
if cfg.TargetContaner != nil {
  targetContainer = aws.StringValue(cfg.TargetContainer)
}
return []ExposedPort{
  {
      Port: targetPort,
      Protocol:      aws.StringValue(protocol),
      ContainerName: targetContainer,
   }
}
```

Reply:
Changing `containerName` to `targetContainer` and `containerPort` to `targetPort`.